MonsterTrainModdingAPI/builders/CardDataBuilder.cs

```diff
@@ -44,6 +46,8 @@ public class CardDataBuilder
         public CardData.CostType CostType { get; set; }
         public FallbackData FallbackData { get; set; }
 
+        public MTClan Clan;
```

Comment:
This should be a property, like all of the ones above it. Just add `{ get; set; }` after it.

Reply:
Fixed